.github/workflows/ci.yml

```diff
@@ -153,7 +153,7 @@ jobs:
       CPPUNIT_IGNORE: class CppUnit::TestCaller<class PathTest>.testFind,class CppUnit::TestCaller<class ICMPSocketTest>.testSendToReceiveFrom,class CppUnit::TestCaller<class ICMPClientTest>.testPing,class CppUnit::TestCaller<class ICMPClientTest>.testBigPing,class CppUnit::TestCaller<class ICMPSocketTest>.testMTU,class CppUnit::TestCaller<class HTTPSClientSessionTest>.testProxy,class CppUnit::TestCaller<class HTTPSStreamFactoryTest>.testProxy
     steps:
       - uses: actions/checkout@v2
-      - run: cmake -S. -Bcmake-build -DENABLE_NETSSL_WIN=ON -DENABLE_NETSSL=OFF -DENABLE_CRYPTO=OFF -DENABLE_JWT=OFF -DENABLE_DATA=ON -DENABLE_DATA_ODBC=ON -DENABLE_DATA_MYSQL=OFF -DENABLE_DATA_POSTGRESQL=OFF -DENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/MP /EHsc" -DCMAKE_C_FLAGS=/MP
+      - run: cmake -S. -Bcmake-build -DENABLE_NETSSL_WIN=ON -DENABLE_NETSSL=OFF -DENABLE_CRYPTO=OFF -DENABLE_JWT=OFF -DENABLE_DATA=ON -DENABLE_DATA_ODBC=ON -DENABLE_DATA_MYSQL=OFF -DENABLE_DATA_POSTGRESQL=OFF -DENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/MP /EHsc /DPOCO_NO_AUTOMATIC_LIBS" -DCMAKE_C_FLAGS="/MP /DPOCO_NO_AUTOMATIC_LIBS"
```

Comment:
@bas524 why `/DPOCO_NO_AUTOMATIC_LIBS` ?

Reply:
Yes, for this build there is no rationale for POCO_NO_AUTOMATIC_LIBS, but when we try build with openssl on windows, we have failed build. 
I think, that it will be more universal with this flag.
But if you say, I'll remove it in next big PR with mswsock.lib

Reply:
If linker is failing to find OpenSSL libs, the root of the problem is either in library names or their location. No automatic libraries is a non-default setting

Reply:
Ok, if you agree, I'll remove this flag with next PR.